src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -225,6 +225,21 @@
   <data name="PropertyNameTooLarge" xml:space="preserve">
     <value>The JSON property name of length {0} is too large and not supported by the JSON writer.</value>
   </data>
+  <data name="Format_Decimal" xml:space="preserve">
+    <value>Value is either too large or too small for a Decimal.</value>
+  </data>
+  <data name="Format_Double" xml:space="preserve">
+    <value>Value is either too large or too small for a Double.</value>
+  </data>
+  <data name="Format_Int32" xml:space="preserve">
+    <value>Value is either of incorrect format or, too large or too small for an Int32.</value>
+  </data>
+  <data name="Format_Int64" xml:space="preserve">
+    <value>Value is either of incorrect format or, too large or too small for an Int64.</value>
+  </data>
+  <data name="Format_Single" xml:space="preserve">
+    <value>Value is either too large or too small for a Single.</value>
```

Comment:
`The JSON value represents a number that is less than 'Single.MinValue' or greater than 'Single.MaxValue'`